whylog/config/parser_name_manager.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+WORDS_COUNT_IN_NAME = 2
+
+
+class ParserNameManager(object):
+    def __init__(self, parsers):
+        self._parsers_name = set()
+        for parser_name in parsers:
+            self._parsers_name.add(parser_name)
+
+    def add_new_parser_name(self, parser_name):
+        self._parsers_name.add(parser_name)
+
+    def is_free_parser_name(self, parser_name, black_list):
+        return (parser_name not in self._parsers_name) and (parser_name not in black_list)
+
+    def _create_name_from_words(self, words, words_count_in_name, black_list):
```

Comment:
Maybe `_create_name_from_words(self, words, black_list, words_count_in_name = WORDS_COUNT_IN_NAME)`:? Or maybe remove this `words_count_in_name` parameter and inside function just use WORDS_COUNT_IN_NAME?
Then propose parser_name will be cleaner.